Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1214.1725849297
```

Comment:
The jira says "empty Dockerfile". Not sure how empty it can really be. But do we really need to copy the whole ubi image? Can it be an image built from scratch? If not, then I would at least recommend ubi minimal image.

Reply:
I am not sure an image "building from scratch" can pass EC.  I forgot ubi-minimal.  Updated!

Reply:
It passed with `FROM scratch`.  Update.

Reply:
cool!